app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -89,3 +105,14 @@
     @include grid-column(12,0);
   }
 }
+
+.main-container--full{
```

Comment:
because this isn't a modifier of .main-container, it should probably be named .main-container-full